src/style.css

```
@@ -0,0 +1,54 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+body {
+  background-color: rgb(247, 247, 247);
+  font-family: "Avenir", sans-serif;
```

move to the `html` selector
```suggestion
  margin: 0;
```